panda/src/device/linuxInputDeviceManager.cxx

```diff
@@ -62,6 +62,9 @@ LinuxInputDeviceManager() {
     // We'll want to sort the devices by index, since the order may be
     // meaningful (eg. for the Xbox wireless receiver).
     std::sort(indices.begin(), indices.end());
+    if(indices.empty()){
```

Comment:
I was thinking this check would go earlier - between lines 60 and 62. std::sort on an empty vector probably isn't expensive but we may as well skip it if it's easy, right?

Also, the if should be spaced out: `if (X) {`